pkg/serverless/tags/tags.go

```diff
@@ -158,6 +162,12 @@ func AddColdStartTag(tags []string, coldStart bool) []string {
 	return tags
 }
 
+// AddSnapStartTag appends the snap_start tag to existing tags
+func AddSnapStartTag(tags []string) []string {
+	tags = append(tags, fmt.Sprintf("snap_start:%v", fmt.Sprint(os.Getenv(InitType) == SnapStartValue)))
```

Comment:
I'm wondering if down the track they add a third kind of init_type, we would have to add another tag. Maybe "init_type:snap_start/on_demand" would be more flexible.

Reply:
That makes sense. We can do a catch-all at this point, since there happens to already be more than the one: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#:~:text=which%20is%20either%20on%2Ddemand%20or%20provisioned%2Dconcurrency